### ISSUE TITLE
Notification on bell?

### ISSUE THREAD

**Issue by logarytm** [gnunn1/tilix#1219]
Using urxvt, I disabled error bells in vim, shell etc. and set `URxvt.bell-command` to run `notify-send` when an alarm arrives (eg. from WeeChat). Can I do it with Tilix without patching VTE?

**Comment by gnunn1** [comment]
It's not possible right now, sorry.

**Comment by egmontkob** [comment]
I think it is possible, VTE generates a "bell" signal which Tilix can catch.

**Comment by logarytm** [comment]
Tilix does catch the "bell" signal, it just cannot send notifications about it yet.

**Comment by egmontkob** [comment]
So I guess it's sloppy wording causing some misunderstanding :-) Tilix _does not_ support this currently, but _could_ support it without patching VTE.

**Comment by gnunn1** [comment]
Yeah I should have bene more explicit. Tilix supports a bunch of options for dealing with the bell however notification isn't one of them.
